app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<main>
+  <div class="recipe-container">
+    <h1 class="recipeTitle"><%= @recipe.name %></h1>
+    <div class="recipeInfo">
+      <div class="recipeInfoL">
+        <p><strong>Preparation Time:</strong> <%= @recipe.preparation_time %> minutes</p>
+        <p><strong>Cooking Time:</strong> <%= @recipe.cooking_time %> minutes</p>
+        <p class="descRecipeD"><%= @recipe.description%><p>
+      </div>
+      <div class="recipeInfoR">
+        <p><strong>Public:</strong></p>
+        <div class="toggle-button">
+          <% if current_user == @recipe.user %>
+            <%= button_to toggle_public_recipe_path(@recipe), method: :patch, class: "btnToggle" do %>
+              <%= @recipe.is_public ? 'Yes' : 'No' %>
+            <% end %>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="buttonsRecipe">
+    <% if current_user == @recipe.user %>
+      <div class="add-food-form">
+        <%= link_to "Add New Food", new_recipe_food_path(@recipe), class: "btn btn-primary" %>
```

Comment:
- [ ] Kindly note that the name of this link is not `add food` it is add `ingrediants` just like in the wireframe

![image](https://github.com/jlvFlores/RoR-recipe-app/assets/65068771/72d1462f-c998-49e3-b70e-a652bbb68052)

![image](https://github.com/jlvFlores/RoR-recipe-app/assets/65068771/a13a9496-57fd-437c-8ba9-9ae764814a3c)